src/components/Editor.js

```diff
@@ -10,6 +10,12 @@ import customCompleter from './customCompleter.js'
 *
 */
 class Editor extends Component {
+  componentWillUnmount() {
+    //Text is saved in state when editor unmounts / when in view only.
+    //This does not fully 'save' it because state will be lost when we 
```

Comment:
Although this is a valid, this is something that's I'm working on fixing. This comment can be simplified to say that it updates the reducer state before closing the editor.